server/src/game/mod.rs

```diff
@@ -331,7 +332,9 @@ impl Game {
 
     pub fn add_grave(&mut self, grave: Grave){
         self.graves.push(grave.clone());
-        OnGraveAdded::new(grave).invoke(self);
+        if let Some(grave_ref) = GraveReference::new(self, self.graves.len() as u8 - 1){
```

Comment:
Could make a `new_unchecked` function like player reference has. Or not, up to you

Reply:
We don't use it so I wont